web/gds-user-ui/src/utils/auth0.helper.ts

```diff
@@ -128,8 +128,10 @@ export const getRefreshToken = async (hasRefreshToken: boolean) => {
 
 export const refreshNewToken = async () => {
   const user = (await refreshAndFetchUser()) as any;
+  console.log('[user?.accessToken]', user?.accessToken);
```

Comment:
Is this meant to be included in the PR? If it is that's fine, just wanted to make sure we're not committing debug code :)

Reply:
no, it's to figure out what is going on in staging. i will remove it later.